Core/Core/Inbox/ViewModel/InboxViewModel.swift

```diff
@@ -135,6 +145,20 @@ public class InboxViewModel: ObservableObject {
             .map { interactor.updateState(message: $0.message, state: $0.state) }
             .sink()
             .store(in: &subscriptions)
+
+        didTapStar
+            .flatMap { [favouriteInteractor] starred, messageId in
+                favouriteInteractor.updateStarred(to: starred, messageId: messageId)
+            }
+            .sink()
+            .store(in: &subscriptions)
+
+        didSendMailSuccessfully
+            .handleEvents(receiveOutput: { [weak self] in
+                self?.snackBarViewModel.showSnack(InboxMessageScope.sent.localizedName)
+            })
```

Comment:
This could go to the sink itself so we could remove this `handleEvents` modifier.